src/VehicleSetup/SetupView.qml

```diff
@@ -110,6 +111,13 @@ Rectangle {
                 panelLoader.sourceComponent = messagePanelComponent
             } else {
                 panelLoader.source = vehicleComponent.setupSource
+                for(var i = 0; i < componentRepeater.count; i++) {
+                    var obj = componentRepeater.itemAt(i);
+                    if (obj.text === vehicleComponent.name) {
+                        obj.checked = true;
```

Comment:
Ah, got it thanks. I wonder if instead of doing this, the showVehicleComponent implementation should do it instead (checking)? Don't like the code being in two different far aways places. Not sure if that is doable or not?